src/Application/UI/Presenter.php

```diff
@@ -1047,7 +1047,7 @@ public static function argsToParams($class, $method, & $args, $supplemental = []
 				));
 			}
 
-			if ($args[$name] === $def || ($def === NULL && is_scalar($args[$name]) && (string) $args[$name] === '')) {
+			if ($args[$name] === $def || ($def === NULL && is_scalar($args[$name]) && !is_bool($args[$name]) && (string) $args[$name] === '')) {
```

Comment:
Probably. Integers and floats should never become an empty string after type casting.